src/mono/wasm/Wasm.Build.Tests/PreloadingTests.cs

```diff
@@ -27,12 +27,11 @@ public void PreloadAssets(bool isPublish, bool fingerprintAssets)
         Configuration config = Configuration.Debug;
         ProjectInfo info = CopyTestAsset(config, aot: false, TestAsset.WasmBasicTestApp, "PreloadAssets");
 
-        // TODO: AssertAppBundle doesn't with fingerprinted dotnet.js at the moment
-        string extraMSBuildArgs = $"-p:OverrideHtmlAssetPlaceholders=true -p:WasmFingerprintAssets={fingerprintAssets}";
+        string extraMSBuildArgs = $"-p:WasmFingerprintAssets={fingerprintAssets}";
         if (isPublish)
-            PublishProject(info, config, new PublishOptions(ExtraMSBuildArgs: extraMSBuildArgs, AssertAppBundle: false));
+            PublishProject(info, config, new PublishOptions(ExtraMSBuildArgs: extraMSBuildArgs, AssertAppBundle: fingerprintAssets));
```

Comment:
I don't understand why we assert only for fingerprinted.

Reply:
I missed we already have `wasmFingerprintDotnetJs`, thank you!

Reply:
Hmm, that probably won't be enough to make AssertAppBundle happy